plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergTestUtils.java

```diff
@@ -57,7 +57,7 @@ public static Session withSmallRowGroups(Session session)
     {
         return Session.builder(session)
                 .setCatalogSessionProperty("iceberg", "orc_writer_max_stripe_rows", "10")
-                .setCatalogSessionProperty("iceberg", "parquet_writer_block_size", "100B")
+                .setCatalogSessionProperty("iceberg", "parquet_writer_block_size", "1kB")
```

Comment:
this is used in file sorting tests.

@alexjo2144 do these tests still test what they were supposed to?

Reply:
Yes, they still do. Earlier we had one row for every row group (which is excessively small), now we have around 10, which is what the intent of the test is. The equivalent ORC test has around 10 rows per stripe as well.
I wonder whether looking at row-group/stripe min-max is the right way to do this. We can read pages directly from the reader and verify sorting at the row level for the resulting pages. Then we wouldn't need to bother about "small" row groups then.

Reply:
It is still valid, if the splits were too large there are assertions that would fail. 

> I wonder whether looking at row-group/stripe min-max is the right way to do this. We can read pages directly from the reader and verify sorting at the row level for the resulting pages. Then we wouldn't need to bother about "small" row groups then

It didn't feel worth doing that in the moment when I was writing these tests but you could be right.